Index: Tier1/WebApplication/WebApplication/Pages/FileTesting.razor
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Tier1/WebApplication/WebApplication/Pages/FileTesting.razor b/Tier1/WebApplication/WebApplication/Pages/FileTesting.razor
new file mode 100644
--- /dev/null	
+++ b/Tier1/WebApplication/WebApplication/Pages/FileTesting.razor	
@@ -0,0 +1,6 @@
+@page "/FileTesting"
+<h3>FileTesting</h3>
+<InputFile>test</InputFile>
+@code {
+    
+}
\ No newline at end of file
